extensions/amp-video/0.1/amp-video.js

```diff
@@ -1034,6 +1038,22 @@ export class AmpVideo extends AMP.BaseElement {
   seekTo(timeSeconds) {
     this.video_.currentTime = timeSeconds;
   }
+
+  /**
+   * Shows or hides the captions.
+   * @param {boolean} captionsOn
+   * @public
+   */
+  toggleCaptions(captionsOn) {
+    toArray(this.video_.textTracks).forEach((track) => {
+      if (captionsOn) {
+        // If captions are delegated then hide (so they show in amp-story-captions), otherwise show.
```

Comment:
This comment is too complex for anyone with no context. What about the captions is delegated? What's amp-story-captions? Why do they need to be hidden?

Example: `// If a custom captions renderer is configured (e.g. amp-story-captions), enable captions but keep them hidden to avoid double rendering.`